### ISSUE TITLE
Apple II disks not loading

### ISSUE THREAD

**Issue by starkis** [TomHarte/CLK#1022]
I just updated to the latest version, and double-clicking an Apple II disk in the Mac Finder isn't loading the disk. A window opens with the disk name, with 'Apple ][' and a command line prompt. If I open our older version of Clock Signal (from July 2020), it works fine, with the game launching automatically. (I'm running macOS Monterey 12.3.1.)

**Comment by TomHarte** [comment]
Confirmed can reproduce; will look into it over the weekend.

**Comment by TomHarte** [comment]
In case it's interesting, this bug relates to one of this emulator's dirty secrets: that it has 90% of a IIgs emulation, which is stalled still not running most software — including the Finder. I strongly suspect an issue in the memory map, since Apple's official documentation is in places slightly ambiguous and no third-party documentation seems to exist; while I was last working on that I inadvertently changed one of the pieces of code shared by both the IIgs and the IIe (related to both being able to page their own ROM over the card address space) so as to cause this breakage.

In short: neither the IIe nor the Enhanced IIe were spotting that they have a Disk II attached, and therefore weren't booting.

Fixed.
